src/Components/ToDoSection/ToDoSection.tsx

```diff
@@ -0,0 +1,16 @@
+import TaskItem from "../TaskItem/TaskItem";
+import styles from "./TodoSection.module.scss";
+
+const TodoSection = () => {
+  return (
+    <div className={styles["todo-section"]}>
```

Comment:
styles.todoSection

Reply:
Okay, if it is the top div in this component should be the classname of that div be from capital letter : styles.TodoSection ?

Reply:
This is a convention we took in our company - if you want to take a different one there you can, just remember to be consistant with it and stick to it. Mainly do not access styles object here by [ ] - this one have use case when you dynamically want to access some class from styles i.e to react on some specific props then you would access it by template string styles[`some-class-${props.someValue}`]

Reply:
I decided for convention of naming top div in component file - 'ComponentName'Component